examples/usbpower/README.rst

```diff
@@ -116,7 +116,7 @@ Remote Setup
 ------------
 
 To access resources remotely, you first need to start the coordinator::
-  $ crossbar start --logformat none --config config-anonymous.yaml
+  $ crossbar-venv/bin/crossbar start --logformat none --config config-anonymous.yaml
```

Comment:
This path doesn't match the one you used above.

Reply:
Which one? `crossbar-venv/bin/crossbar` is used everywhere except in the release step-by-step guide.